lang/lang_en_hu.txt

```diff
@@ -1,1403 +1,1403 @@
 
 #MSG_WIZARD_Z_CAL c=20 r=8
 "I will run z calibration now."
-"\x00"
+"Lefuttatom a Z kalibraciot."
 
 #MSG_WATCH c=18
 "Info screen"
-"\x00"
+"Info kepernyo"
 
 #MSG_INSERT_FILAMENT c=20
 "Insert filament"
-"\x00"
+"Helyezd be a filam."
 
 #MSG_FILAMENT_LOADED c=20 r=2
 "Is filament loaded?"
-"\x00"
+"Filament behelyezve?"
 
 #MSG_STEEL_SHEET_CHECK c=20 r=2
 "Is steel sheet on heatbed?"
-"\x00"
+"Rajta van az acellap a targyasztalon?"
 
 #MSG_LAST_PRINT_FAILURES c=20
 "Last print failures"
-"\x00"
+"Utolso nyomt. hibak"
 
 #MSG_WIZARD_WELCOME_SHIPPING c=20 r=16
 "Hi, I am your Original Prusa i3 printer. I will guide you through a short setup process, in which the Z-axis will be calibrated. Then, you will be ready to print."
-"\x00"
+"Szia, en vagyok az eredeti Prusa i3 nyomtatod. Vegigvezetlek egy rovid beallitasi folyamaton, amelyben a Z tengelyt kalibralom. Utana mar nyomtathatsz is."
 
 #MSG_ADDITIONAL_SHEETS c=20 r=9
 "If you have additional steel sheets, calibrate their presets in Settings - HW Setup - Steel sheets."
-"\x00"
+"Ha tobb acellapod van, kalibrald be oket a Beallitasok - HW beall. - Acellapok menupont alatt."
 
 #MSG_LAST_PRINT c=18
 "Last print"
-"\x00"
+"Utolso nyomtatas"
 
 #MSG_SELFTEST_EXTRUDER_FAN c=20
 "Left hotend fan?"
-"\x00"
+"Bal hotend vent.?"
 
 #MSG_LEFT c=10
 "Left"
-"\x00"
+"Bal"
 
 #MSG_BED_CORRECTION_LEFT c=14
 "Left side [um]"
-"\x00"
+"Bal [um]"
 
 #MSG_LIN_CORRECTION c=18
 "Lin. correction"
-"\x00"
+"Lin. korrekcio"
 
 #MSG_BABYSTEP_Z c=18
 "Live adjust Z"
-"\x00"
+"Z magassag beall."
 
 #MSG_INSERT_FIL c=20 r=6
 "Insert the filament (do not load it) into the extruder and then press the knob."
-"\x00"
+"Helyezd be a filamentet az extruderbe (ne toltsd be), majd nyomtd meg a gombot."
 
 #MSG_LOAD_FILAMENT c=17
 "Load filament"
-"\x00"
+"Filament betolt."
 
 #MSG_LOADING_COLOR c=20
 "Loading color"
-"\x00"
+"Szin tisztitasa"
 
 #MSG_LOADING_FILAMENT c=20
 "Loading filament"
-"\x00"
+"Filament betoltese"
 
 #MSG_ITERATION c=12
 "Iteration"
-"\x00"
+"Iteracio"
 
 #MSG_LOOSE_PULLEY c=20
 "Loose pulley"
-"\x00"
+"Laza szijtarcsa"
 
 #MSG_LOAD_TO_NOZZLE c=18
 "Load to nozzle"
-"\x00"
+"Betolt. fuvokahoz"
 
 #MSG_M117_V2_CALIBRATION c=25
 "M117 First layer cal."
-"\x00"
+"M117. Elso reteg kalibr."
 
 #MSG_MAIN c=18
 "Main"
-"\x00"
+"Fomenu"
 
 #MSG_BL_HIGH c=12
 "Level Bright"
-"\x00"
+"Fenyes szint"
 
 #MSG_BL_LOW c=12
 "Level Dimmed"
-"\x00"
+"Sotet szint"
 
 #MSG_MEASURE_BED_REFERENCE_HEIGHT_LINE1 c=20 r=3
 "Measuring reference height of calibration point"
-"\x00"
+"Kalibracios pont magassaganak merese"
 
 #MSG_MESH_BED_LEVELING c=18
 "Mesh Bed Leveling"
-"\x00"
+"Asztal szintezes"
 
 #MSG_MMU_OK_RESUMING_POSITION c=20 r=4
 "MMU OK. Resuming position..."
-"\x00"
+"MMU OK. Pozicio visszaallitasa..."
 
 #MSG_MMU_OK_RESUMING_TEMPERATURE c=20 r=4
 "MMU OK. Resuming temperature..."
-"\x00"
+"MMU OK. Homerseklet visszaallitasa..."
 
 #MSG_MEASURED_SKEW c=14
 "Measured skew"
-"\x00"
+"Meroleg. hiba"
```

Comment:
@AttilaSVK it nem lenne inkab job "Mert ferdeseg" ez csak javaslat :)

Reply:
A merőlegességi hiba pontosabb megfogalmazás szerintem.

Reply:
> A merőlegességi hiba pontosabb megfogalmazás szerintem.
@AttilaSVK igen azert montam hogy csak javaslat

Reply:
After you agree on a translations, please comment it in English so we can read it as well.

Reply:
@3d-gussner will do in the future.
Regarding this translation, instead of skew I used the term perpendicularity error. (it sounds better in Hungarian IMHO, and it’s easier to shorten nicely)